grid.css

```diff
@@ -6,8 +6,10 @@
   padding-right: var(--gutter-width, 1rem);
 
   @media (--lg-viewport) {
-    padding-left: calc(6 * var(--gutter-width, 1rem));
-    padding-right: calc(6 * var(--gutter-width, 1rem));
+    padding-left: calc(10 * var(--gutter-width, 1rem));
+    padding-right: calc(10 * var(--gutter-width, 1rem));
+    max-width: 90rem;
+    width: 100%;
```

Comment:
`max-width` is cool; I'm still iffy about the `width`, but just merge now and I'll pull later and play with it in a browser.